android/app/src/main/java/com/dismoi/scout/Floating/Module.java

```diff
@@ -0,0 +1,230 @@
+package com.dismoi.scout.Floating;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Bundle;
+import android.provider.Settings;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ImageButton;
+
+import androidx.annotation.NonNull;
+
+import com.dismoi.scout.Floating.Layout.Bubble;
+import com.dismoi.scout.Floating.Layout.Message;
+import com.dismoi.scout.Floating.OnCallback;
+import com.dismoi.scout.R;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+
+public class Module extends ReactContextBaseJavaModule {
```

Comment:
No that I see it a second time `Module` I suppose it's some kind of Java pattern :sweat_smile:

Reply:
No is my pattern :D

Reply:
After giving it more thoughts, it'd feel more natural to name the class as the name used in `getName`.
It's presented this way in `react-native` doc https://reactnative.dev/docs/native-modules-android#module-name
But again do as you feel more comfortable with.

Reply:
Ok I didn't see and remember this part of the doc. Will apply